build_rules.gradle

```diff
@@ -67,6 +67,9 @@ repositories {
   maven { url "https://repository.apache.org/content/repositories/releases" }
 }
 
+// Help IntelliJ - specific hints are in their natures
```

Comment:
Update comment to:
`Apply a plugin which enables configuring projects imported into Intellij.`

Reply:
Done